resources/Components/Countdown/index.js

```diff
@@ -0,0 +1,19 @@
+import useCountdown from '../../Hooks/useCountdown'
+import { ClusterRow, ClusterValue } from '../Cluster'
+export default Countdown = ({ end, handleClick, title }) => {
+  const ttl = useCountdown(end)
+  return (
+    <ClusterRow>
```

Comment:
what does all this cluster stuff do? are we just wrapping every component w. this?

Reply:
I think it probably makes sense to add these wrapper element outside of this component as this component could be used in other contexts